### PR TITLE
remove deprecated go vet tool fetch in Dockerfile

### PR DESCRIPTION
**Pull request by PI-Victor**
currently the project fails to build.
go vet tool has been removed and included in go, it's no longer accessible at: https://godoc.org/golang.org/x/tools/cmd

### CODE REVIEW

**Comment by mssola** [comment]
Tests are failing. Could you look into that ? Moreover, maybe we could add more Go versions, but let's leave that for another PR 😉

File: helpers.go [hunk 1]
```diff
@@ -200,10 +201,10 @@ func preventImageOverwrite(repo, tag string) error {
 	imageExists, err := checkImageExists(repo, tag)
 
 	if err != nil {
-		return fmt.Errorf("Cannot proceed safely: %v.", err)
+		return fmt.Errorf("cannot proceed safely: %v", err)
 	}
 	if imageExists {
-		return fmt.Errorf("Cannot overwrite an existing image. Please use a different repository/tag.")
+		return errors.New("cannot overwrite an existing image. Please use a different repository/tag")
```
Comment (tboerger) [inline helpers.go]: Why `errors.New`? Just use `fmt.Errorf` like it's done above. You just need to drop the dot at the end of the error message.

Comment (PI-Victor) [inline helpers.go]: this is just a nit. fmt.Errorf is used for formatting, there's no formatting done in this error, hence errors.new.

Comment (tboerger) [inline helpers.go]: But even without a format value it works fine, IMHO it should not be mixed. Use entirely `errors.New` or `fmt.Errorf`.

Comment (flavio) [inline helpers.go]: We never discussed our coding conventions, so far AFAIK we always used `fmt.Errorf`.

I would stick with `fmt.Errorf` even in this case. What do the others like @mssola and @inercia think about that?

Comment (mssola) [inline helpers.go]: We are pretty inconsistent for this to be honest, but if we follow the documentation, `errors.New` is the proper function. That being said, I don't have any strong feelings, and I think that this change is unrelated to the PR anyways ...

Comment (tboerger) [inline helpers.go]: Yeah this pr should only drop the dot at the end of the message :)

Comment (PI-Victor) [inline helpers.go]: @tboerger not sure why you got hung up on this. i think things should be used by the book and not according to personal pref. 

closing this. not worth the effort.

Comment (tboerger) [inline helpers.go]: I just responded to @mssola, the linting just complained about the dot at the end of the error message
